### PR TITLE
Proper JSON for facia-tool archiving

### PR DESCRIPTION
**Pull request by janua**
After #8015, the `Json` was being put into `Dynamo` as a weird escaped string. This fixes that, and puts it into the DB as proper `dyanmo` `json`.

This adds conversions from `JsValue` to `AttributeValue` that is accepted by `PutItemRequest` via the `JsValue` catamorphism. I opted for using implicit classes instead of using the strange `Read` syntax.

I had to bump the version of `aws-java-sdk` in order to get methods such as `withM` and `withL`.
#### Sidenote: Json and Aws-java-sdk

It is really unfortunate that you can't easily put `Json` into `Dynamo` in an `asynchronous` manner. 
`Item` from the bumped version supports `fromJSON(s: String)`, but it is useless as the `AmazonDynamoDbAsyncClient` can't handle `Item`s. The only way to use `Item`s directly is to get a `Table` instance and then use `putItem` with that which seems to be completely blocking.

What's worse is that `PutItemSpec` supports adding `Item`s, but when you use `getRequest()` on it, you get back a `PutItemRequest` with the internal `items` set to `null`. So basically, it's useless.

@robertberry

### CODE REVIEW

File: common/app/implicits/JsonImplicits.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    private def jsArray(seq: Seq[JsValue]): AttributeValue = new AttributeValue().withL(seq.map(_.fold(jsNull, jsBool, jsNumber, jsString, jsArray, jsObject, jsUndefined)): _*)
+    private def jsObject(obj: Seq[(String, JsValue)]): AttributeValue = new AttributeValue().withM(
+      obj.map { case (s, json) => s -> json.fold(jsNull, jsBool, jsNumber, jsString, jsArray, jsObject, jsUndefined)}.toMap.asJava)
+    private def jsUndefined: AttributeValue = new AttributeValue()
+
+    def toAttributeValue: AttributeValue = {
+      jsValue.fold(jsNull, jsBool, jsNumber, jsString, jsArray, jsObject, jsUndefined)
+    }
+  }
+}
```
Comment (robertberry) [inline common/app/implicits/JsonImplicits.scala]: Nice.

**Comment by robertberry** [comment]
That sucks re blocking. I sometimes feel an urge to write a pure Scala DynamoDB client, as the Java one is yuck in a bunch of ways. Laziness has meant I've not so far, though.

**Comment by robertberry** [comment]
:+1:

**Comment by janua** [comment]
Yeah, it's bad. Something doesn't seem right with the `aws-java-sdk` at times; in this instance is has two abstractions that are sort of the same but don't operate with each other.

Needs to be a lovely scala lib for it!

**Pull request merged**